docs/reference/snapshot-restore/apis/create-snapshot-api.asciidoc

```diff
@@ -0,0 +1,122 @@
+[[create-snapshot-api]]
+=== Create index snapshot API
+++++
+<titleabbrev>Create index snapshot</titleabbrev>
+++++
+
+Takes a snapshot of one or more indices. A repository can contain multiple snapshots of the same cluster. Snapshots are identified by unique names in the cluster, such as `snapshot_1`.
+
+////
+[source,console]
+-----------------------------------
+PUT /_snapshot/my_backup
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+-----------------------------------
+// TESTSETUP
+////
+
+[source,console]
+-----------------------------------
+PUT /_snapshot/my_backup/snapshot_1?wait_for_completion=true
+-----------------------------------
+
+[[create-snapshot-api-request]]
+==== {api-request-title}
+
+`POST /_snapshot/<repository>/<snapshot-name>`
+
+[[create-snapshot-api-desc]]
+==== {api-description-title}
+
+By default, a snapshot of all open and started indices in the cluster is created. You can change this behavior by specifying the list of indices in the body of the snapshot request.
+
+NOTE: You must register a snapshot before performing snapshot and restore operations. Use the <<put-snapshot-repo-api,put snapshot repository API>> to register new repositories and update existing ones.
+
+The index snapshot process is incremental. When creating the index snapshot, {es} analyzes the list of index files that are already stored in the repository and copies only files that were created or changed since the last snapshot. This process allows multiple snapshots to be preserved in the repository in a compact form.
```

Comment:
I made some suggestions to remove index references. These also apply to data streams now.
```suggestion
The snapshot process is incremental. When creating a  snapshot, {es} analyzes the list of files that are already stored in the repository and copies only files that were created or changed since the last snapshot. This process allows multiple snapshots to be preserved in the repository in a compact form.
```